Geometry/HGCalCommonData/test/python/g4OverlapCheckWafer_cfg.py

```diff
@@ -3,7 +3,7 @@
 
 process = cms.Process("OverlapTest",Phase2C11)
 
-process.load('Geometry.HGCalCommonData.testHGCalWaferTestXML_cfi')
+process.load('Geometry.HGCalCommonData.testHGCalWaferXML_cfi')
```

Comment:
After changing, do you still need `testHGCalWaferTestXML_cfi` ?

Reply:
This is testing the earlier cell version. It is good to preserve it till the version V12 is preserved